### ISSUE TITLE
Bug: VQE algorithm-design/cost_function code block error for optimizing max-cut-vqe

### ISSUE THREAD

**Issue by david-alber** [Qiskit/platypus#2097]
### Where does the issue happen?

https://learn.qiskit.org/course/algorithm-design/cost_function

### What is the content issue?

**Problem + cause:**
The code cell, see screenshot, fails. Using the minimize method of the COBYLA optimizer to evaluate the cost_function_max_cut_vqe() requires an initial point ``x0`` as input. 

**How to solve:**
Update the [cost_function.ipynb](https://github.com/Qiskit/platypus/tree/main/notebooks/algorithm-design/cost_function/) notebook to use an initial point ``initial_point = np.ones(8)``.

```
initial_point = np.ones(8)
optimizer_result = optimizer.minimize(fun=cost_function_max_cut_vqe,x0=initial_point)
```

### Screenshots

![Screenshot 2023-05-03 at 14 46 04](https://user-images.githubusercontent.com/47004675/235920726-fb75539e-a3a0-4cf5-92dd-b4e306854c5e.png)

**Comment by david-alber** [comment]
I'd love to pick up this issue!

**Comment by david-alber** [comment]
The issue has been resolved by [PR #2098](https://github.com/Qiskit/platypus/pull/2098/commits/2ceb3542ef01de53f10514a3ee659018b1e9b87f). Thanks @pandasa123.
